python/lsst/summit/utils/guiders/detection.py

```diff
@@ -303,17 +303,18 @@ def runSourceDetection(
     exposure = ExposureF(MaskedImageF(ImageF(image)))
 
     # Step 2: Detect sources
-    footprints = detectObjectsInExp(exposure, nSigma=threshold)
+    # we assume that we have bright stars
+    # filter out stamps with no stars
+    if not isBlankImage(image, fluxMin=200):
```

Comment:
Is there a reason you make the default 500, but override it here?

Reply:
I was playing around with the value. You can leave the default value 500.